types/glint.d.ts

```diff
@@ -12,6 +12,7 @@ declare module '@glint/environment-ember-loose/registry' {
     capitalize: HelperLike<{ Return: string; Args: { Positional: [string] } }>;
     'date-format': HelperLike<{ Return: string; Args: { Positional: [Date]; Named: { format?: string } } }>;
     'date-from-now': HelperLike<{ Return: string; Args: { Positional: [Date]; Named: { currentDate?: Date } } }>;
+    'format-duration': HelperLike<{ Return: string; Args: { Positional: [number] } }>;
```

Comment:
This file is used for helpers that we don't control (from libraries) - for helpers in our app, we place the glint registrations directly in the file like this: 

https://github.com/codecrafters-io/frontend/blob/fd9d5570c1082fd6b1a0825d3b290c2e826b5487/app/helpers/current-user-is-course-author.ts#L25-L29

By doing this, we don't have to specify the interface separately, it is inferred from the function. This prevents mistakes where the interface specified here doesn't match the actual helper.
 
Let's move this registration into the helper file, and let's also move `date-format` and `date-from-now` into files please - they were previously part of a library we were using and were converted to helpers later. All the other helpers in this file _should_ be from libraries.